form.go

```diff
@@ -318,13 +325,17 @@ func (f *Form) WithHeight(height int) *Form {
 }
 
 // WithOutput sets the io.Writer to output the form.
+// Default is STDOUT when [Form.Accessible], STDERR otherwise.
```

Comment:
this field isn't public, so this link doesn't work. 
```suggestion
// Default is STDOUT when [Form] is accessible (set with [Form.WithAccessible], STDERR otherwise.
```